diff --git a/CMakeLists.txt b/CMakeLists.txt
index 5aecafc..7e44d50 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -25,12 +25,13 @@ if(${CMAKE_BUILD_TYPE} STREQUAL Debug)
 endif(${CMAKE_BUILD_TYPE} STREQUAL Debug)
 
 # find libusb
-find_package ( USB1 REQUIRED )
-include_directories ( ${LIBUSB_INCLUDE_DIR} )
+#find_package ( USB1 REQUIRED )
+#include_directories ( ${LIBUSB_INCLUDE_DIR} )
 
 # Find Boost
 if (FTDIPP OR BUILD_TESTS)
   find_package( Boost REQUIRED )
+  include_directories(${CONAN_INCLUDE_DIRS_BOOST})
 endif()
 
 # Set components
@@ -57,7 +58,7 @@ if ( NOT DEFINED LIB_SUFFIX )
       AND NOT EXISTS "/etc/debian_version"
       AND NOT EXISTS "/etc/arch-release" )
     if ( "${CMAKE_SIZEOF_VOID_P}" EQUAL "8" )
-      set ( LIB_SUFFIX 64 )
+      set ( LIB_SUFFIX "" )
     endif ()
   endif ()
 endif ()
diff --git a/ftdipp/CMakeLists.txt b/ftdipp/CMakeLists.txt
index fac5bcc..435ee72 100644
--- a/ftdipp/CMakeLists.txt
+++ b/ftdipp/CMakeLists.txt
@@ -12,6 +12,7 @@ include_directories(BEFORE ${CMAKE_CURRENT_BINARY_DIR}
 include_directories(${Boost_INCLUDE_DIRS})
 
 # Shared library
+if ( NOT STATICLIBS )
 add_library(ftdipp1 SHARED ${cpp_sources})
 
 math(EXPR VERSION_FIXUP "${MAJOR_VERSION} + 1")    # Compatibility with previous releases
@@ -28,6 +29,7 @@ install ( TARGETS ftdipp1
 					LIBRARY DESTINATION lib${LIB_SUFFIX}
 					ARCHIVE DESTINATION lib${LIB_SUFFIX}
 				)
+endif ()
 
 # Static library
 if ( STATICLIBS )
diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index 17b3617..d2d554d 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -20,7 +20,7 @@ configure_file(ftdi_version_i.h.in "${CMAKE_CURRENT_BINARY_DIR}/ftdi_version_i.h
 # Targets
 set(c_sources     ${CMAKE_CURRENT_SOURCE_DIR}/ftdi.c ${CMAKE_CURRENT_SOURCE_DIR}/ftdi_stream.c CACHE INTERNAL "List of c sources" )
 set(c_headers     ${CMAKE_CURRENT_SOURCE_DIR}/ftdi.h CACHE INTERNAL "List of c headers" )
-
+if ( NOT STATICLIBS )
 add_library(ftdi1 SHARED ${c_sources})
 
 math(EXPR VERSION_FIXUP "${MAJOR_VERSION} + 1")    # Compatibility with previous releases
@@ -37,7 +37,7 @@ install ( TARGETS ftdi1
           LIBRARY DESTINATION lib${LIB_SUFFIX}
           ARCHIVE DESTINATION lib${LIB_SUFFIX}
         )
-
+endif ()
 if ( STATICLIBS )
   add_library(ftdi1-static STATIC ${c_sources})
   target_link_libraries(ftdi1-static ${LIBUSB_LIBRARIES})
